apps/financial-aid/backend/src/environments/environment.ts

```diff
@@ -29,6 +29,18 @@ const prodConfig = {
     postTimeToLiveMinutes: 5,
     getTimeToLiveMinutes: 5,
   },
+  email: {
+    fromEmail: process.env.EMAIL_FROM ?? '',
+    fromName: process.env.EMAIL_FROM_NAME ?? '',
+    replyToEmail: process.env.EMAIL_REPLY_TO ?? '',
+    replyToName: process.env.EMAIL_REPLY_TO_NAME ?? '',
```

Comment:
What will be the values here on different envs?

Reply:
I am also wondering if this should be in environment. Because the end product will have different values depending on municipalities which we will read from a table (probably) so maybe this should just be hardcoded until we get there. Any thoughts?

Reply:
Ahh good point! I didn't realize, we have different email for municipality well spotted, I will hardcoded until we get there 🤖